src/server/lock_manager/config.rs

```diff
@@ -12,6 +12,7 @@ use tikv_util::config::ReadableDuration;
 #[serde(default)]
 #[serde(rename_all = "kebab-case")]
 pub struct Config {
+    // Deprecated. The enable flag was only for compatible issue from v3.0 or older version.
     #[config(skip)]
     pub enabled: bool,
```

Comment:
I think we can just remove `enabled`.

Reply:
Ok!

Reply:
😳What if an old version config file contains it? I guess TiKV will report an error.

Reply:
There won't be an error, Serde should just ignore it